maintenance/projects/js-themes-toolkit-v8-x/packages/liferay-theme-tasks/test/util.js

```diff
@@ -321,7 +321,7 @@ function cleanTempTheme(themeName, version, component, initCwd) {
 
 	cleanDirectory(tempPath);
 
-	if (initCwd != null) {
+	if (initCwd !== null && initCwd !== undefined) {
```

Comment:
Can't we express this as `if (initCwd)` . I find `if (initCwd !== null && initCwd !== undefined)` a bit too complex to use it in every place of the code...

Reply:
My goal with using `initCwd !== null && initCwd !== undefined` is that it is safer for a wide refactor since `initCwd != null` is equal to `initCwd !== null && initCwd !== undefined`

If I was writing new code, I would likely use `if (initCwd)`, but I can't totally be sure that this bit of code would work with that unless I tested it thoroughly(which I didn't since I touched lots of code).

Reply:
>  If it is only for this PR, it's OK and we may refactor them when we come back to the code. But if it is going to be our usual pattern, I find it a bit too much for my refined taste

Just saw this from below. Yep, this is just for this PR and not something I intend for us to do. Only made that change for some safety in refactoring.